src/types.ts

```diff
@@ -1,4 +1,4 @@
 export type FormatterName = 'branch' | 'commit' | 'command';
-export type stringConverterFn = (input: string) => string;
-/* eslint-disable @typescript-eslint/no-explicit-any */
-export type parseFn = (values?: any) => string;
+export type StringConverterFn = (input: string) => string;
```

Comment:
Maybe `StringTransformFn` or `StringMappingFn` would be a better name? Maybe it's just in my head, but I associate "convert" with something changing type, e.g. "converting matter into energy". 🤔

Reply:
`StringMappingFn` sounds great, thanks for suggesting a better name 💚

Reply:
ah, implemented it, but in a new PR (which I'm about to prepare) to not further complicate this PR with more topics :)